### DISCUSSION TITLE
下载的Minecraft的执行文件在哪？

### DISCUSSION THREAD

**Discussion by J1npen** [Meloong-Git/PCL#4801] [提问] [answered]
因为我想在启动Minecraft时为我的鼠标侧键设置成“ESC”，但是设置前提是要知道Minecraft的执行程序（.exe）文件的地址。
希望能回答一下，Thanks(･ω･)ﾉ

**Comment by 3gf8jv4dv** [comment, marked as answer]
Minecraft Java Edition is written in Java, so it needs to be launched using the Java environment installed on your PC and does not have a standalone executable program. You can find all the Java environments you have in PCL’s "Settings → Game → Launch Options → Java", select one of them, and specify it in the program you use.

Minecraft（Java 版）是使用 Java 编写的，它需要用你电脑上已安装的 Java 环境进行启动，并没有独立的可执行程序。你可以在 PCL 的 “设置 → 游戏 → 启动选项 → 游戏 Java” 找到你所拥有的所有 Java 环境，选择其中一个，并在你使用的程序中指定它。

**Reply by J1npen** [reply]
好的，谢谢🖐
